src/Apps/Artwork/Utils/insightUtils.ts

```diff
@@ -0,0 +1,9 @@
+export const formatSellThroughRate = (sellThroughRate: number | null) => {
```

Comment:
**note:** I added a ticket to add this to metaphysics in the future and update it in Eigen.

Reply:
Reading my mind! Let's merge these fixes right now and add the field to MP in the next step.